packages/dds/sequence/src/sharedString.ts

```diff
@@ -299,3 +301,38 @@ export function getTextAndMarkers(sharedString: SharedString, label: string): {
 } {
     return sharedString.getTextAndMarkers(label);
 }
+
+interface SharedStringInternals {
+    intervalCollections: DefaultMap<IntervalCollection<SequenceInterval>>;
+}
+
+export class TestSharedString
```

Comment:
Here and for basically all of the other added classes that you're using to inject the behavior of saving using the old format: it would be better to move them to a file inside the test folder, so that we're sure we don't affect end-users' bundle sizes. We don't want consumers loading this code (or really even seeing it).

Reply:
After you move things around, I'd also rename to reflect the purpose of the class. Here you're trying to make the SharedString write interval collections using the old format, so something like `SharedStringWithV1IntervalCollection` seems more appropriate. It's wordy, but basically everywhere other than where you construct the class you shouldn't care (you can just use the SharedString type)